vars/redhat-8.yml

```diff
@@ -14,5 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-_yum:
-  - { state: "latest", name: "firefox" }
+_yum: []
```

Comment:
EPEL Testing have package for CentOS/RHEL 8, see https://centos.pkgs.org/8/epel-testing-x86_64/gnucash-4.1-1.el8.x86_64.rpm.html#howto

Need the `enablerepo` additional parameters